config/jobs/kubernetes-sigs/node-feature-discovery/node-feature-discovery-postsubmits.yaml

```diff
@@ -59,3 +68,10 @@ postsubmits:
+        resources:
```

Comment:
TODO: evaluate the necessary amount for this job (happy to defer that and get it migrated first)

Reply:
for our e2e we have tried 2 cpu's but it runs out of time. 4C/8M has been the min set , but happy to evaluate if we can trim down a bit